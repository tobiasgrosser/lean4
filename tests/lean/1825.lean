theorem boom : -2147483648 + 2147483648 = 0 := by rfl
theorem boom' : -2147483648 + 2147483648 = -18446744069414584320 := by native_decide
