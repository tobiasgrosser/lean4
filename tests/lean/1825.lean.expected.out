1825.lean:2:8-2:13: error: application type mismatch
  Lean.ofReduceBool boom'._nativeDecide_1 true (Eq.refl true)
argument has type
  true = true
but function has type
  Lean.reduceBool boom'._nativeDecide_1 = true â†’ boom'._nativeDecide_1 = true
